pkg/umorse/Makefile.dep

```diff
@@ -0,0 +1,3 @@
+ifneq (,$(filter umorse,$(USEPKG)))
+  USEMODULE += posix_time
```

Comment:
actually I didn't want to have this dependency in uMorse, but rather use xtimer_usleep if used with RIOT. However, it seems I need to adapt uMorse a bit for that, too. We can either merge this first and revert/fix it later on, or wait and I fix uMorse RIOTs integration first and omit this change completely.

Reply:
Since this PR is about the introduction of the `posix_time` and not about fixing `umorse`'s dependency to that, let's do it this and you can then adapt `umorse` in a separate PR (though I prefer it this way. This is why the POSIX wrappers exist in the first place)

Reply:
mhm, yeah the posix to xtimer wrapper here are slim - I'll think about it ...

Reply:
So can we merge as is for now?